### ISSUE TITLE
Standardise Metric Naming

### ISSUE THREAD

**Issue by DanTanAtAims** [open-AIMS/ADRIA.jl#743]
Coral Cover metric naming should be standard across ADRIA. 
- `absolute_cover`: coral cover in absolute units, e.g., m^2
- `relative_cover`:  coral cover relative to **habitable** area. Ideally in [0, 1]
- `total_cover`: coral cover relative to total reef area including **uninhabitable**. Ideally in [0, 1]

Interfaces loading external data sets should make sure the naming is assigned to the correct data and that any unit conversions are applied.

**Comment by ConnectedSystems** [comment]
Note for everyone: this will be part of #465, and could be part of the refactor of all metrics to leverage YAXArrays/metadata (#524).

**Comment by Zapiano** [comment]
I think there might be a confusion when using "absolute" and "total". The word "relative" gives a hint that the cover will be a fraction of something, but it doesn't say "relative to what". Since we have cover relative to the habitable area and relative to the total area, what do you think about `relative_habitable_cover` and `relative_total_cover`? Too much?

**Comment by arlowhite** [comment]
I agree with including "habitable" in the name. `relative_habitable_cover` 👍 
maybe "all" instead of "total"? so `relative_all_cover`

**Comment by VHallerBull** [comment]
I agree with Arlo, both would be considered relative just to a different reference. 

The other part to consider is resolution, from my use of adria so far it seems to use total_cover when there is one value per scenario while the other two are used for one value per location (so hundreds of timeseries per scenario). So, the question here is, do we use the same names independent of the resolution of should we have differences i.e.
`loc_relative_cover` and `relative_cover` would be the same metric but at the two different resolutions

**Comment by ConnectedSystems** [comment]
> so far it seems to use total_cover when there is one value per scenario

You should be able to use any metric summarised by scenario, location. Less supported are the functional group-based metrics.

**Comment by ConnectedSystems** [comment]
Further to the above, the metrics are documented (although how it's presented needs work)

https://open-aims.github.io/ADRIA.jl/dev/API/#Metrics
